src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/TestFiles/RazorSourceGeneratorComponentTests/ScriptTag/Views_Home_Index.html

```diff
@@ -0,0 +1,11 @@
+﻿1: <script>alert('hello')</script>
+2: <script>alert(&#x27;hello&#x27;)</script>
+3: <div>alert('hello')</div>
+4: <script>
+alert('hello')</script>
+Component:
+1: <script>alert('hello')</script>
+2: <script>alert(&#x27;hello&#x27;)</script>
+3: <div>alert('hello')</div>
+4: <script>
+alert('hello')</script>
```

Comment:
Yes, this output looks correct to me. I'm assuming the fact that `<script>alert('hello')</script>` on line 1 works is what's changed with this PR, and that it would have emitted the same as line 2 before this PR. Is that right?

AFAICT what you have here are the desired outputs.

Reply:
> I'm assuming the fact that `<script>alert('hello')</script>` on line 1 works is what's changed with this PR, and that it would have emitted the same as line 2 before this PR. Is that right?

Yes, you can see that in the diff between the two commits in this PR: https://github.com/dotnet/razor/pull/9241/commits/53188d6dd60d1dbe7f6971dff1287a3f54b088ae#diff-40d01947571b2bce53da7c09f965aaf1706265577f5d86d9888a9b74192fe53b